LKMEX-whitelist/combased-custom

```diff
@@ -0,0 +1,29 @@
+## Name of the project
+Comverse
+
+## Description of the project
+Combased project is web3 agency, a leader in e-commerce space & core DeFi protocol creator. Read more here https://combased.io
+
+## Purpose
+We send to our holders monthly rewards with LKME.
+
+To farm XMEX on the dex we use our CDO wallet, which takes 50% of all royalties and earnings from our services. We provide web3 development services, JewelSwap and sell our Shopify apps.
+
+To qualify all eligible addresses we use our first open source script, which was actually instrumental in developing Elrond ecosystem.
+
+https://github.com/Combased/elrond-lkmex-distribution
```

Comment:
maybe change the repo, also? Remove elrond?

Reply:
Updated, yes, removed :)